src/main/java/cyano/mineralogy/Mineralogy.java

```diff
@@ -111,25 +94,26 @@
 	private static final String fertilizer = "fertilizer";
 	private static final String stone = "stone";
 	private static final String dustCarbon = "dustCarbon";
-	private static final String blockPhosphorous = "blockPhosphorous";
-	private static final String dustPhosphorous = "dustPhosphorous";
+	private static final String blockPhosphorous = "phosphorous_block";
```

Comment:
HUGE redflag comes up in my mind on this change, these Strings contained oredicts, not block names.
It's looking like you're not passing block names where oredicts should have been passed,

Reply:
thanks, I'll sort this out later